fbchat/client.py

```diff
@@ -88,11 +88,6 @@ def __init__(self, email, password, debug=True, user_agent=None):
     def _console(self, msg):
         if self.debug: print(msg)
 
-    def _setttstamp(self):
```

Comment:
Sorry, my earlier message was vague.

My main concern with this function is that it reads and writes to global state for no reason. It being a separate function is not my biggest concern.

As a secondary concern, however, if you would like to keep it a function, I suggest moving the function inside the `_generatePayload` function (I believe this is what I did in the PR), so that we are not exposing more functions than is necessary. This makes it easier for a newbie to read and understand your code, because they can easily tell that the function is only used inside `_generatePayload`.